From a598e0b48183d64b52df53c556f02b53969f8ad8 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io> [Modified by Thomas Staudinger <Staudi.Kaos@gmail.com>]
Date: Sat, 26 Dec 2020 19:41:58 +0200
Subject: [PATCH 1/1] Support a stateless configuration.

- Remove /usr/etc since in no circumstance would that ever be supported in Solus.
- Add /usr/share/alsa/conf.d as an option to enable us to move stuff out of /etc/alsa/conf.d
---
 src/conf/alsa.conf | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/conf/alsa.conf b/src/conf/alsa.conf
index a04173a..f0c29f5 100644
--- a/src/conf/alsa.conf
+++ b/src/conf/alsa.conf
@@ -8,10 +8,10 @@
 	{
 		func load
 		files [
-			"/var/lib/alsa/conf.d"
-			"/usr/etc/alsa/conf.d"
 			"/etc/alsa/conf.d"
-			"/etc/asound.conf|||/usr/etc/asound.conf"
+			"/etc/asound.conf"
+			"/usr/share/defaults/alsa/asound.conf"
+			"/usr/share/alsa/conf.d"
 			"~/.asoundrc"
 			{
 				@func concat
-- 
2.29.2

